tools/ds-identify

```diff
@@ -281,6 +280,49 @@ read_fs_info() {
     DI_ISO9660_DEVS="${isodevs#,}"
 }
 
+read_geom() {
+    local oifs="$IFS" line="" delim=","
+    local ret=0 out="" labels="" dev="" label="" ftype="" isodevs="" uuids=""
```

Comment:
check if geom is available?
command -v geom

Reply:
it's in `/sbin/geom`

which means it's in "base".
i don't even see a `WITHOUT_GEOM`: https://www.freebsd.org/cgi/man.cgi?src.conf
so i don't think you can build a modern FreeBSD without geom